pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml

```diff
@@ -250,9 +301,25 @@ spec:
                       description: Tags is a map of key/value tags used to select
                         subnets Specifying '*' for a value selects all values for
                         a given tag key.
+                      maxProperties: 30
```

Comment:
Same comment as amiSelectorTerms. I'd like to see us be more restrictive on the number of tags and less restrictive on the number of terms